src/typescript.ts

```diff
@@ -634,47 +633,35 @@ export class TypescriptConfig {
 }
 
 
-class SampleCode extends Component {
-  private readonly nodeProject: TypeScriptProject;
-
+class SampleCode {
```

Comment:
Should still inherit from `Component`:

```suggestion
class SampleCode extends Component {
```